Packages/ClientRuntime/Sources/Waiters/JMESValue.swift

```diff
@@ -0,0 +1,92 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A value type that is used to facilitate comparison of values returned by JMESPath types, in accordance with the JMESPath spec.
+public enum JMESValue: Equatable, Comparable {
```

Comment:
Nit: imo it'd be great if we didn't need to expose this publicly since its an implementation detail of the waiters acceptors and instead we just publicly exposed some static function on Acceptors or something. Especially since we just need this type to perform comparisons.

Reply:
Removed in favor of several overloaded static functions.  See JMESUtils.swift.